README.md

```diff
@@ -4,7 +4,7 @@
 [![shield_rtfd]][link_rtfd]
 [![shield_pypi-version]][link_pypi]
 [![shield_docker-version]][link_docker]
-[![shield_license]][license_file]  
```

Comment:
:x: 
the two white spaces were there for a reason. they are line breaks in markdown.
please revert!